packaging/darwin/amazon-cloudwatch-agent-ctl

```diff
@@ -65,6 +65,8 @@ UsageString="
             file:<file-path>:                       file path on the host
             all:                                    all existing configs. Only apply to remove-config action.
 
+        -v: Return exit code 99 if validating agent's json config fails.
```

Comment:
This is probably a nitpick, but I disagree very strongly about the flag name we are choosing for this. Typically, `-v` is reserved for "verbose" or "version". I don't think that we should use this flag name for strict validation. Is there any way we can use a longer name than a single character? I would like this to just be `-strict` if possible.

Reply:
Agree on this part. -v stands for version or verbose. However, if using `long command`, current `getopts` would only suitable for single character. Therefore, utilizing `https://github.com/UrsaDK/getopts_long` to provide addition long arguments for current flags and `strict` flag

Reply:
I'm open to other suggestions though. I would err towards just using a different character rather than pulling in another library just to allow using a long name `-strict`. @ecerulm suggested this so would love feedback from them on what they envision it could/should be. 

I think one thing we could do is try to find examples of other established tools that have some form of fail-fast or strict validation mechanism to see what the consensus is for the naming of the flag used for it.

Reply:
That's also another thing I am stucking when inventing another character for `--strict`, thought of change `s` from restart to strict but it's a default behavior for CWAgent entire time (kinda one way door a little bit).

Reply:
Yeah, I wouldn't change the meaning of another flag. Like I said, I think we should look at other software for potential ideas for reasonable flags that we can use here for the naming of this feature flag. Personally, I thought maybe `-f` and change the meaning to `--failfast` or something like that rather than just thinking of some variation of `strict-validation`. I think it's worth taking the time to look at other examples for what makes the most sense here before committing.